docs/en/Building-Alluxio-Master-Branch.md

```diff
@@ -59,68 +59,71 @@ To have the logs output to STDOUT, append the following to the `mvn` command
 {% include Building-Alluxio-Master-Branch/STDOUT.md %}
 
 ## Compute Framework Support
-To build Alluxio against different compute frameworks, you can run Maven build with different compute profiles. The generated Alluxio client is located at `{{site.ALLUXIO_CLIENT_JAR_PATH}}`.
 
-### Hadoop
+Since Alluxio 1.7, the Alluxio client jar built and located at
+`{{site.ALLUXIO_CLIENT_JAR_PATH}}` will work with different compute frameworks
+(e.g., Spark, Flink, Presto and etc). **You do not need run Maven build with different compute
+profiles.**
 
-You can run the following command to compile Alluxio with Hadoop.
+## Hadoop Distribution Support
 
-```bash
-$ mvn install -P<HADOOP_PROFILE> -DskipTests
-```
-
-Available Hadoop profiles include `hadoop-1`, `hadoop-2.2`, `hadoop-2.3` ... `hadoop-2.8` for
```

Comment:
should we remove these profiles from the root pom?

Reply:
will do in a separate pr